cli/cfncluster/cfnconfig.py

```diff
@@ -228,7 +228,7 @@ def __init__(self, args):
                                       cwl_log_group=('CWLLogGroup',None),shared_dir=('SharedDir',None),tenancy=('Tenancy',None),
                                       ephemeral_kms_key_id=('EphemeralKMSKeyId',None), cluster_ready=('ClusterReadyScript','URL'),
                                       master_root_volume_size=('MasterRootVolumeSize',None),compute_root_volume_size=('ComputeRootVolumeSize',None),
-                                      base_os=('BaseOS',None),ec2_iam_role=('EC2IAMRoleName','EC2IAMRoleName'),extra_json=('ExtraJson',None),
+                                      base_os=('BaseOS',None),ec2_iam_role=('EC2IAMRoleName', 'EC2IAMRoleName'),extra_json=('ExtraJson',None),
```

Comment:
remove space from this line or add space between (x, y) in the mapping, just to be consistent

Reply:
done.